integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -147,22 +147,38 @@ public int chown(String path, @uid_t long uid, @gid_t long gid) {
           path, PropertyKey.FUSE_USER_GROUP_TRANSLATION_ENABLED.getName());
       return -ErrorCodes.ENOSYS();
     }
+
+    boolean isUserSet = true;
+    if (uid == -1 || uid == 4294967295L) {
```

Comment:
I think it's part of the fuse API. in my local mac and ubuntu linux, uid is always `4294967295` when not provided user name. There may not be a case that it will be `-1`. I will remove the `-1` check. Thanks